db/migrate/20150929152345_move_arf_reports_to_reports_table.rb

```diff
@@ -0,0 +1,179 @@
+class MoveArfReportsToReportsTable < ActiveRecord::Migration
+
+  # rubocop:disable Metrics/MethodLength
+  # rubocop:disable Metrics/AbcSize
+  def up
+    old_arf_reports = execute("SELECT * FROM foreman_openscap_arf_reports;")
+
+    #select only reports with existing host
+    old_arf_reports = old_arf_reports.select do |item|
+      asset = ForemanOpenscap::Asset.find item['asset_id']
+      !asset.host.nil? && asset.assetable_type = "Host::Base"
+    end
+    #and remove assets without assetable
+    ForemanOpenscap::Asset.where(:assetable_type => "Host::Base").select { |a| a.host.nil? }.map(&:destroy)
+    ForemanOpenscap::Asset.where(:assetable_type => "Hostgroup").select { |a| a.hostgroup.nil? }.map(&:destroy)
+
+    old_arf_reports.each do |item|
+      metrics = breakdown_to_metrics item["id"]
+
+      #reported_at attribute must be unique
+      reported_at = DateTime.strptime(item["created_at"], "%Y-%m-%d %H:%M:%S")
+
+      reported_at += 1.seconds until arfs_by_reported(reported_at).empty?
+
+      arf = ForemanOpenscap::ArfReport.create!(:metrics => metrics,
+                                               :reported_at => reported_at,
+                                               :created_at => item["created_at"],
+                                               :updated_at => item["updated_at"],
+                                               :host_id => item["asset_id"],
+                                               :status => metrics)
+
+      ForemanOpenscap::PolicyArfReport.create!(:arf_report_id => arf.id, :policy_id => item["policy_id"], :digest => item["digest"])
+
+      xccdf_rules.each { |rule_item| Source.find_or_create(rule_item["xid"]) }
+
+      xccdf_rule_results(item["id"]).each do |rr_item|
+        message = Message.find_or_create("No message for this log")
+
+        rule_item = xccdf_rule(rr_item['xccdf_rule_id'])
+        source = Source.find_or_create(rule_item['xid'])
+
+        Log.create!(:report_id => arf.id,
+                    :result => xccdf_result(rr_item["xccdf_result_id"])['name'],
+                    :message_id => message.id,
+                    :source_id => source.id,
+                    :level => :info)
```

Comment:
@xprazak2 - I am not sure. I feel that the "info" level is irrelevant in our use case. 
Either we drop it from the view, or map our severity to existing levels.
@ares - what do you think?